raiden-ts/src/epics.ts

```diff
@@ -99,20 +101,37 @@ export function getLatest$(
     ),
     startWith({} as Latest['rtc']),
   );
-  // the nested combineLatest is needed because it can only infer the type of 6 params
-  return combineLatest([
-    combineLatest([action$, state$, config$, presences$, pfsList$, rtc$]),
-    combineLatest([udcBalance$]),
-  ]).pipe(
-    map(([[action, state, config, presences, pfsList, rtc], [udcBalance]]) => ({
-      action,
-      state,
-      config,
-      presences,
-      pfsList,
-      rtc,
-      udcBalance,
-    })),
+
+  return using(
+    // using will ensure these subscriptions only happen at output's subscription time
+    // and are properly teared down upon output's unsubscribe, complete or error
```

Comment:
I actually fail to come up with an idea how to express what this does (as you explained me) via code. So even this is RxJS definition I would like to extend this comment to state that we add a child subscription, that also automatically gets unsubscribed if the parent does.